server/azure-local-service/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "azure-local-service",
+  "version": "0.4.0",
```

I think this needs to rev as it's own pipeline to keep parity with the azure-client package. 

```suggestion
  "version": "0.1.0",
```